TestAssets/TestProjects/TestProjectToProjectDependencies/src/L21/project.json

```diff
@@ -1,10 +1,10 @@
 {
   "version": "1.0.0-*",
   "dependencies": {
-    "NETStandard.Library": "1.5.0-rc2-24018"
+    "Microsoft.NETCore.App": "1.0.0-rc2-*"
   },
   "frameworks": {
-    "netstandardapp1.5": {
+    "netcoreapp1.0": {
```

Comment:
If this is causing problems for your PR, you can feel free to ignore or log a separate issue for this comment.  It was wrong before (using netstandardapp), so you aren't making it any worse by targeting netcoreapp.